pages/entropy/generate-random-numbers/evm.mdx

```diff
@@ -200,3 +161,20 @@ Check the [Current Fees](../current-fees) to find the current fee for each provi
 ### Best Practices
 
 Check out the [Best Practices](../best-practices) guide for tips to limit gas usage, or generate multiple random numbers in a single transaction.
+
+<Callout type="info">
+Entropy also requires selecting a **randomness provider**. The randomness provider is a third-party
+who participates in the generation process. Each provider is identified by an address and hosts
+a keeper service for fullfilling requests.
+
+The simplest way to choose a provider is to use the [default provider](../contract-addresses).
```

Comment:
and the best way to get this is to call the `getDefaultProvider()` method on the contract. It's much better to use the method than hardcode the address (what if we change the provider??)